interfaces/cython/cantera/ctml_writer.py

```diff
@@ -798,6 +809,27 @@ def build(self, t):
         u["size"] = "9"
         u["name"] = "coeffs"
 
+class standardState(object):
+    """Base class for species standard-state properties."""
+    def _build(self, p):
+        return p.addChild("standardState")
```

Comment:
Does this method get called anywhere? All I see is the commented-out call from `constantIncompressible.build`.

Reply:
Is it called in `species.build`, lines 651-652?

Reply:
I see a call to `build` in `species.build`, but not a call to `_build`.

Reply:
Okay, hitting the limits of my knowledge, here, but is the `standardState` class needed at all?